reconcile/ocm_clusters.py

```diff
@@ -75,9 +87,10 @@ def run(dry_run, gitlab_project_id=None, thread_pool_size=10):
     error = False
     clusters_updates = {}
     for cluster_name, desired_spec in desired_state.items():
+        logging.info(f"Processing cluster {cluster_name}")
```

Comment:
Would this log unconditionally on each execution, or am I missing something? Basically wondering if this would be extra logging in Slack that isn't necessarily associated with a change?